### PR TITLE
Add JSX docs view to all components

### PR DESCRIPTION
**Pull request by nicko-winner**
Also removed storybook docs now that we are hosting storybook, if we wanted to bring them back maybe it would make more sense to add that information to a CONTRIBUTING.md.

### CODE REVIEW

**Review by 100stacks — CHANGES_REQUESTED** [review]
@nicko-winner, just a few small comments/questions.  👍

File: README.md [hunk 1]
```diff
@@ -118,27 +116,6 @@ Each wrapper corresponds with a component listed on the [Helix UI documentation]
...
-2. Run Storybook
-
-```bash
-npm start
-```
-The script should automatically open Storybook in your browser, but you can always navigate to `http://localhost:6006/?path=/story`
-
-3. Play Around
-<img width="639" alt="Screen Shot 2020-05-15 at 2 32 35 PM" src="https://user-images.githubusercontent.com/22566333/82084224-f5c99880-96b8-11ea-92e2-403b3f4378db.png">
-
```
[the remark this answers is not in the document]
Comment (nicko-winner) [reply README.md:141]: sure

File: src/Alert/stories.js [hunk 2]
```diff
@@ -1,8 +1,13 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
+import { addParameters, storiesOf } from '@storybook/react';
 import { boolean, select, text } from '@storybook/addon-knobs/react';
 import Alert from './index';
 import { action } from '@storybook/addon-actions';
+import { callback } from '../storyUtils';
+
+addParameters({
+  jsx: { skip: 0 },
```
[the remark this answers is not in the document]
Comment (nicko-winner) [reply src/Alert/stories.js:9]: This creates a global param, and sets the arguments for storybook. We are telling the JSX plugin that is being applied globally to all stories to skip: 0 times, which means, do not exclude any markup from the JSX window display:

If you had markup that looks like this:
skip: 0 would would show all the markup.
```
<div>
     <span>
        <a></a>
     </span>
</div>
```

skip: 1. would skip the outermost div:
```
<span>
     <a></a>
</span>
```
skip: 2 would skip the outer 2 tags:
```
  <a></a>
```

File: src/Button/stories.js [hunk 3]
```diff
@@ -2,9 +2,13 @@ import Button from './';
 import { Icons } from '../Icon';
 import React from 'react';
 import centered from '@storybook/addon-centered/react';
-import { storiesOf } from '@storybook/react';
+import { addParameters, storiesOf } from '@storybook/react';
 import { boolean, select, text } from '@storybook/addon-knobs/react';
 
+addParameters({
+  jsx: { skip: 2 },
```
[the remark this answers is not in the document]
Comment (nicko-winner) [reply src/Button/stories.js:9]: see earlier response.

File: src/Alert/stories.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (100stacks) [reply src/Alert/stories.js:9]: Ok thanks for the explanation above. 🙄

**Review by 100stacks — APPROVED** [review]
Dev LGTM

**Pull request merged**